cmake/podioMacros.cmake

```diff
@@ -0,0 +1,95 @@
+#---------------------------------------------------------------------------------------------------
+#---REFLEX_GENERATE_DICTIONARY( dictionary headerfiles SELECTION selectionfile OPTIONS opt1 opt2 ...
```

Comment:
```suggestion
#---PODIO_GENERATE_DICTIONARY( dictionary headerfiles SELECTION selectionfile OPTIONS opt1 opt2 ...
```
This comment refers to `PODIO_GENERATE_DICTIONARY` or should remind the use of `REFLEX...`?

Reply:
Thanks, I modified this line as well. Or did you mean something else?

Reply:
No that's all I meant. Thanks!